indico/queries/documents.py

```diff
@@ -80,6 +81,11 @@ def __init__(
         self.files = files
         self.json_config = json_config
         self.upload_batch_size = upload_batch_size
+        for f in self.files:
```

Comment:
I'm concerned with this check, this assumes all the pdf will have .pdf at the end which might not be the case in all customer envs. It's a very crude way to check.

Reply:
What do you think about something like below? The filetype.match(f) would potentially lead here to a FileNotFoundError- but I think that's probably better than also having that happen later as well. 
```
        for f in self.files:
            doctype = filetype.match(f)
            if doctype.MIME not in ('application/pdf',):
                raise IndicoError(
                    "Only PDF files can be used with DocumentExtraction. You provided: {f}"
                )
```

Reply:
yeah that should work, as long as it's consistent on windows and unix hopefully

Reply:
Shouldn't this work with TIF files as well?  Should also check ".tif", ".tiff"